src/Achievements/AchievementsKit.cs

```diff
@@ -53,6 +55,7 @@ protected override void Instantiate(Address initialManager)
 			{
 				{ "Admin", initialManager.ToString() },
 				{ "User", AccessControlList.AnyAddress },
+				{ "Achievements", achievements },
```

Comment:
Do we really need to pass in an empty list? We can just make the parameter optional